src/NuGet.Core/NuGet.PackageManagement/BuildIntegration/BuildIntegratedProjectAction.cs

```diff
@@ -52,7 +53,56 @@ public class BuildIntegratedProjectAction : NuGetProjectAction
             IReadOnlyList<SourceRepository> sources,
             IReadOnlyList<NuGetProjectAction> originalActions,
             BuildIntegratedInstallationContext installationContext)
-            : base(packageIdentity, nuGetProjectActionType, project)
+            : base(packageIdentity, nuGetProjectActionType, project, null)
+        {
```

Comment:
As mentioned in another comment, it's best to only have a single constructor with a method body, and all constructor overloads call into it. It's much easier to make sure there's no bugs when there's only one.